secboot/secboot_sb.go

```diff
@@ -397,9 +400,18 @@ func RenameKeys(node string, renames map[string]string) error {
 	return nil
 }
 
+// RenameContainerKey renames a key slot on LUKS2 container. An error
+// is returned if cryptsetup does not support --token-replace option.
```

Comment:
is there an explicit error value returned in this case?

Reply:
yes, https://github.com/canonical/snapd/blob/982e0cb88bec921e709c7b2bffaa98f049ecb0c4/secboot/secboot_sb.go#L385-L393

although I saw this warning to not use it for new code, that's why I dropped the fallback bits
https://github.com/canonical/snapd/blob/982e0cb88bec921e709c7b2bffaa98f049ecb0c4/secboot/secboot_sb.go#L354-L356